audformat/core/utils.py

```diff
@@ -1000,21 +1000,20 @@ def union(
         MultiIndex([('f1', '0 days 00:00:00', '0 days 00:00:01'),
                     ('f2', '0 days 00:00:00', '0 days 00:00:01'),
                     ('f3', '0 days 00:00:00', '0 days 00:00:01'),
-                    ('f3', '0 days 00:00:01', '0 days 00:00:02'),
-                    ('f4', '0 days 00:00:00', '0 days 00:00:01')],
+                    ('f4', '0 days 00:00:00', '0 days 00:00:01'),
+                    ('f3', '0 days 00:00:01', '0 days 00:00:02')],
                    names=['file', 'start', 'end'])
-
         >>> union([index1, index2, index3, index4])
         MultiIndex([('f1', '0 days 00:00:00',               NaT),
-                    ('f1', '0 days 00:00:00', '0 days 00:00:01'),
-                    ('f2', '0 days 00:00:00',               NaT),
-                    ('f2', '0 days 00:00:00', '0 days 00:00:01'),
-                    ('f3', '0 days 00:00:00',               NaT),
-                    ('f3', '0 days 00:00:00', '0 days 00:00:01'),
-                    ('f3', '0 days 00:00:01', '0 days 00:00:02'),
-                    ('f4', '0 days 00:00:00',               NaT),
-                    ('f4', '0 days 00:00:00', '0 days 00:00:01')],
-                   names=['file', 'start', 'end'])
+                ('f2', '0 days 00:00:00',               NaT),
+                ('f3', '0 days 00:00:00',               NaT),
```

Comment:
Interesting, did it ignore the extra space of the return values in the test before?

Reply:
It now does look strange to me:

![image](https://user-images.githubusercontent.com/173624/151127326-70fbf9d8-1932-46f0-810b-b9d0fa81c8c4.png)

When I execute it I get:

```python
>>> union([index1, index2, index3, index4])
MultiIndex([('f1', '0 days 00:00:00',               NaT),
            ('f2', '0 days 00:00:00',               NaT),
            ('f3', '0 days 00:00:00',               NaT),
            ('f4', '0 days 00:00:00',               NaT),
            ('f1', '0 days 00:00:00', '0 days 00:00:01'),
            ('f2', '0 days 00:00:00', '0 days 00:00:01'),
            ('f3', '0 days 00:00:00', '0 days 00:00:01'),
            ('f4', '0 days 00:00:00', '0 days 00:00:01'),
            ('f3', '0 days 00:00:01', '0 days 00:00:02')],
           names=['file', 'start', 'end'])
```

Could you try to change include the spaces again?

Reply:
Indeed, it seems to be quite relaxed with leading white spaces in the output. I now changed it again to. I re-added the missing spaces, though.

Reply:
Thanks, looks good now in the docs.